go/enclave/components/consumer_test.go

```diff
@@ -1,28 +1,20 @@
-package l2chain
+package components
 
 import (
 	"math/big"
 	"testing"
 
-	"github.com/obscuronet/go-obscuro/go/common/log"
-
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/trie"
-
-	gethlog "github.com/ethereum/go-ethereum/log"
 )
 
 func TestInvalidBlocksAreRejected(t *testing.T) {
-	// There are no tests of acceptance of valid chains of blocks. This is because the logic to generate a valid block
-	// is non-trivial.
-	genesisJSON, err := core.DefaultGenesisBlock().MarshalJSON()
-	if err != nil {
-		t.Errorf("could not parse genesis JSON: %v", err)
-	}
-	logger := log.New(log.DeployerCmp, int(gethlog.LvlDebug), log.SysOut)
-	chain := ObscuroChain{l1Blockchain: NewL1Blockchain(genesisJSON, logger)}
+	t.Skipf("skipping test that relied on l1blockchain previously which hasn't been used in a while")
```

Comment:
worth deleting the test or you want to leave it so we can replace it with a similar one?